src/Rel8/Table/Either.hs

```diff
@@ -151,15 +153,6 @@ instance
       err = error "Either.fromColumns: mismatch between tag and data"
 
 
-instance
-  ( Reifiable from, from ~ from'
-  , Reifiable to, to ~ to'
```

Comment:
I thought we were done with `Reifiable` - is the fact that we still have a `Reifiable` type an accident, or are we still actually using it?

Reply:
We're not done with `Reifiable`, but `Reifiable` is merely means `KnownContext`.

Reply:
I've been thinking of moving/renaming the module it resides in but I've had bigger fish to fry. Feel free to do that yourself though if you think there's a better name